starfish/pipeline/pipelinecomponent.py

```diff
@@ -7,6 +8,10 @@ class PipelineComponentType(type):
     This is the metaclass for PipelineComponent.  As each subclass that is _not_ PipelineComponent is created, it sets
     up a map between the algorithm name and the class that implements it.
     """
+
+    # this seems to be expected by this class as well.
+    _algorithm_to_class_map: Optional[Mapping[str, Type]] = None
```

Comment:
that's pycharm being confused.  it shouldn't be required.

Reply:
Got it. I'll take that out.

Reply:
Oh, you need to rebase, I added this already.